game/utils/map.py

```diff
@@ -0,0 +1,148 @@
+class AbstractMap:
+    """
+    Abstract class for the map.
+    """
+
+    population = (int,)
+    generation = (int,)
+    map = (list[list[bool]],)
+
+    def __init__(self):
+        """
+        Initializes the map with a default map containing living cells on coordinates y = 1 and x from 0 to 4
+        and a population of 5.
+        """
+        self.population = 5
+        self.generation = 0
+
+        blank_map = [[False for _ in range(10)] for _ in range(10)]
+        blank_map[1][0] = True
+        blank_map[1][1] = True
+        blank_map[1][2] = True
+        blank_map[1][3] = True
+        blank_map[1][4] = True
+
+        self.map = blank_map
+
+    def __str__(self) -> str:
+        """
+        Converts the map to a string.
+        :return: Map as a string.
+        """
+        map_to_str = ""
+        for row in self.map:
+            map_to_str += "".join(["#" if cell else "." for cell in row]) + "\n"
+
+        return map_to_str
+
+    def count_neighbours(self, x, y) -> int:
+        """
+        Counts the number of living cells around a cell.
```

Comment:
Documentation like this quickly becomes outdated, doesn't add anything important to the table and just makes code looks worse